src/working-mode

```diff
@@ -0,0 +1,242 @@
+   being removed from implementations.
+
+   <li>
+    <p>A test ensuring the feature is not supported must be added and existing tests for the feature
+    should be adjusted or removed as appropriate.
+
+    <p class="example"><a href="https://github.com/w3c/web-platform-tests/pull/5001">w3c/web-platform-tests#5001</a>
+    added new tests that appcache was not supported in shared workers, per the removal in
+    <a href="https://github.com/whatwg/html/pull/2384">whatwg/html#2384</a>.
+
+    <p class="example">Adjusting existing tests can be difficult. If necessary, an issue can be
+    filed instead to track updating those tests. This was done in
+    <a href="https://github.com/w3c/web-platform-tests/issues/5053">w3c/web-platform-tests#5053</a>
+    which accompanied the removal in
+    <a href="https://github.com/whatwg/html/pull/2402">whatwg/html#2402</a>.
+  </ul>
+
+  <h3>New Proposals</h3>
+
+  <p>As described above, the criteria for inclusion in a WHATWG standard is rather strict. In the
+  initial stages of feature development for the web platform, such widespread implementer support is
+  often not available, or the shape of a feature is not yet clear enough for implementers to feel
+  comfortable pledging their support.
+
+  <p>In such scenarios we anticipate features will be "incubated" outside of WHATWG standards. This
+  could be in a variety of venues, such as:
+
+  <ul>
+   <li><p>Another standards organization
+   <li><p>A personal document or GitHub repository
+   <li><p>A pull request to a WHATWG Standard, with the understanding it will not be merged until
+   the criteria above are met
+   <li>
+    <p>A document hosted by the WHATWG, but not under the spec.whatwg.org subdomain and not titled
+    "Living Standard"
+
+    <p class="example"><a href="https://wiki.whatwg.org/wiki/CanvasColorSpace">CanvasColorSpace</a>.
+  </ul>
+
+  <p>In all cases, features that hope to graduate to a WHATWG standard should strive to follow the
+  above guidelines, gather appropriate implementer commitments, and have corresponding tests.
+
+  <p>Additionally, those maintaining such proposals should try to involve the WHATWG community, for
+  example by filing an issue on the standard they anticipate eventually becoming a part of.
+
+  <p class="note">This process of incubation can be very lightweight, such as filing a pull request
+  on the appropriate WHATWG standard with a proposal, and gathering appropriate implementer support
+  there. Or it could be more involved, such as creating a separate repository at another venue and
+  iterating for a long time there.
+
+  <p class="example">The addition of <code>URL.prototype.toJSON()</code> was rather straightforward
+  once a plan was agreed upon and upstream IDL issues were resolved:
+  <a href="https://github.com/whatwg/url/issues/137">whatwg/url#137</a>.
+
+  <p class="example"><code>innerText</code> was initially drafted in a repository outside the HTML
```

Comment:
Maybe link "a repository outside the HTML Standard" to https://github.com/rocallahan/innerText-spec/